src/coreclr/vm/eetwain.cpp

```diff
@@ -2394,3 +2277,60 @@ ULONG32 EECodeManager::GetStackParameterSize(EECodeInfo * pCodeInfo)
 #endif // TARGET_X86
 }
 
+#ifdef FEATURE_INTERPRETER
+
+bool InterpreterCodeManager::UnwindStackFrame(PREGDISPLAY     pContext,
+                                              EECodeInfo     *pCodeInfo,
+                                              unsigned        flags,
+                                              CodeManState   *pState)
+{
+    // Interpreter-TODO: Implement this
```

Comment:
Is there a specific reason why these aren't `ASSERTE(false)` like the other interpreter TODOs?

Reply:
I didn't add assert to the ones that I knew we need to implement and that I am already implementing in a follow up change to enable stack walking with interpreted frames on stack. For the ones with the assert, those are the ones that I have thought that won't likely be implemented for the interpreter.